sale_order_line_date/models/sale_order.py

```diff
@@ -14,7 +14,9 @@ class SaleOrder(models.Model):
 
     @api.onchange("commitment_date")
     def _onchange_commitment_date(self):
-        """Update empty commitment date order lines with commitment date from sale order"""
+        """
+        Update empty commitment date order lines with commitment date from sale order
+        """
```

Comment:
This change should be into "pre-commit commit" -> [[IMP] sale_order_line_date: pre-commit auto fixes](https://github.com/OCA/sale-workflow/pull/3091/commits/17e716ba9930a05a2f0f032fa29457075f849618)